app/assets/stylesheets/user.scss

```diff
@@ -58,3 +29,34 @@
   text-align: center;
 }
 
+.big-error {
+  color: $google-red;
+  font-size: 1em;
+}
+
+.small-error {
+  @extend .pull-right;
+  color: $google-red;
+  font-size: 0.8em;
+}
+
+.forgot-password {
+  #show-password:checked + [for="show-password"]::before {
+    border-right: 2px solid $my-black;
+    border-bottom: 2px solid $my-black;
+    -moz-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+  }
+}
+
+.sign-up-container {
+  box-shadow: 5px 5px 5px #888888;
```

Comment:
Color literals like `#888888` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered border-radius, box-shadow, padding, position, top, width